src/GraphQL.Authorization.Tests/GraphQL.Authorization.Tests.csproj

```
@@ -1,7 +1,7 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFrameworks>netcoreapp2.2</TargetFrameworks>
+    <TargetFrameworks>netcoreapp3.1;netcoreapp3.0;netcoreapp2.2</TargetFrameworks>
```

```suggestion
    <TargetFrameworks>netcoreapp3.1</TargetFrameworks>
```

Multitargeting not needed here. You may just change single target to the latest netcore version.